<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>881</width>
      <height>291</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>         国内、国际/中国港澳台     </string>
     </attribute>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>96</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>单程</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>96</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>往返</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>多程（含缺口程）</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>80</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(153, 153, 153);
border: none;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>出发地</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
font-weight: bold;
font-size: 14px;</string>
       </property>
       <property name="placeholderText">
        <string>可输入城市或机场</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(153, 153, 153);
border: none;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>目的地</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
font-weight: bold;
font-size: 14px;</string>
       </property>
       <property name="placeholderText">
        <string>可输入城市或机场</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(153, 153, 153);
border: none;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>出发日期</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
font-weight: bold;
font-size: 14px;</string>
       </property>
       <property name="placeholderText">
        <string>可输入城市或机场</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>80</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(153, 153, 153);
border: none;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>返回日期</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
font-weight: bold;
font-size: 14px;</string>
       </property>
       <property name="placeholderText">
        <string>可输入城市或机场</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>80</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(153, 153, 153);
border: none;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>乘客类型</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
font-weight: bold;
font-size: 14px;</string>
       </property>
       <property name="placeholderText">
        <string>可输入城市或机场</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>160</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="new_resources.qrc">:/newimages/newimages/1.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="new_resources.qrc">:/newimages/newimages/3.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="new_resources.qrc">:/newimages/newimages/2.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>20</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>不限舱等</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>经济舱</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>公务/头等舱</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>        特价机票       </string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>          航班动态        </string>
     </attribute>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>96</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>搜航班号</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>96</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>搜起降地</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(153, 153, 153);
border: none;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>出发日期</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
font-weight: bold;
font-size: 14px;</string>
       </property>
       <property name="placeholderText">
        <string>可输入城市或机场</string>
       </property>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>59</y>
        <width>591</width>
        <height>71</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>581</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>  请填写航班号，如MU1234</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
     <zorder>stackedWidget</zorder>
     <zorder>radioButton_4</zorder>
     <zorder>radioButton_5</zorder>
     <zorder>widget_7</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>              在线选座           </string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>          退票改签            </string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>      更多服务    </string>
     </attribute>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>报销凭证</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>机场攻略</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>国内机场大全</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>国际机场大全</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>团体票</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;|&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;我的订单&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;联系客服&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="new_resources.qrc"/>
 </resources>
 <connections/>
</ui>
